repos/system_upgrade/el7toel8/actors/satellite_upgrade_check/libraries/satellite_upgrade_check.py

```diff
@@ -28,7 +28,9 @@ def satellite_upgrade_check(facts):
         This will happen automatically during the first boot of the system.
         """).strip()
 
-        if facts.postgresql.same_partition:
+        if not facts.postgresql.scl_pgsql_data:
+            migration_msg = "Your PostgreSQL data seems to be already migrated."
```

Comment:
Nit: Will it make sense to add "No additional action will be taken" to provide more clarity to the user?

Reply:
Yes and no ;-)
An action *will* be taken: the reindex will happen. But you are right, no *movement* of the data will take place, which is a good thing to point out! Will add!